modules/core/src/main/java/org/apache/fluo/core/impl/FluoConfigurationImpl.java

```diff
@@ -101,6 +101,44 @@ public static int getTxCommitMemory(FluoConfiguration conf) {
     }
     return m;
   }
+  
+  // Note the size of the cache is in megabytes
+  public static final String TX_INFO_CACHE_SIZE = FLUO_IMPL_PREFIX + ".tx.failed.cache.size.mb";
+  /* TODO: What is a reasonable default size? Currently CacheBuilder does not specify a size in TxInfoCache.java */
```

Comment:
It indirectly specifies a size by specifying at weight and weigher.  A weigher could return anything, in this case it returns size. The TxStatusWeigher returns the size + 24 (there is not coment, but I assume the 24 is some estimated overhead).  Currently its specifies a max weight of 10,000,000, so that would be a good default.    Need to look for other config that has a size in mb and see if multiplies by 10^6 or 2^20, it would be good to be consistent.

Reply:
I did some digging and found three precedents.  In FluoConfigurationImpl we have a commit memory which has a unit of bytes.  In FluoYarnConfig we have `fluo.yarn.worker.max.memory.mb` these end up being passed to Twill and it seems that Twill has a SizeUnit class.  However the Twill SizeUnit class does not document if MEGA is 10^6 or 2^20.  Based on past experience I think it may 2^20, but not sure.  I would say multiply by 2^20 for now.